Src/Notion.Client/Models/Page/CreatedPage.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Collections.Generic;
+using System.Runtime.Serialization;
+
+namespace Notion.Client
+{
+    [DataContract]
+    public class CreatedPage
```

Comment:
@asevos do you feel we can have builder class and keep the properties as public - so it will be users choice how they want to create the instance.

does it make sense to rename this class to `NewPage`?

Reply:
Renamed class. Not sure about making properties public, as this might create confusion about the correct way of working with class, and eventually force developer to read docs instead of figuring it out from the way class works. However, I made them public and added constructor with no arguments so initialization with class literal would be available.